global.json

```diff
@@ -1,5 +1,5 @@
 {
     "msbuild-sdks": {
-        "Microsoft.Quantum.Sdk": "0.16.2104138281-beta"
```

Comment:
It would be good to set all version numbers to the same value across the runtime repo. Could you please ensure that this version number matches the number used in the project file, and then do a search-and-replace for all version numbers in this repo?

Reply:
I found only 3 files:
* [global.json](https://github.com/microsoft/qsharp-runtime/pull/675/files#diff-8df3cd354bc584349d04ad5675b33c042d8b99b741b8b95af394c55e0f5001bf)
* [Microsoft.Quantum.CSharpGeneration.fsproj](https://github.com/microsoft/qsharp-runtime/pull/675/files#diff-ef47fd9dc8d21524bbdd440cc3ba7816393dc7cfd8a3725b56b0728984af2c97)
* [Microsoft.Quantum.Xunit.nuspec](https://github.com/microsoft/qsharp-runtime/pull/675/files#diff-a163f985a6bea4c07f3ad76766279e56ba7ce897f25e694e0e03e9c23949c9c9)

and changed in all three.

Reply:
There are more, that use the Sdk. I suggest to open the entire src folder in the repo root e.g. with VS Code and do a search and replace.

Reply:
My bad, the global.json replaces them.